app/scripts/views/settings/clients.js

```diff
@@ -46,7 +47,6 @@ define(function (require, exports, module) {
         });
       }
 
-      this.listenTo(this._attachedClients, 'add', this._onItemAdded);
```

Comment:
We always call `render` when refreshing the attachedClients list anyways, no need to re-render because of this. re-rendering every time an item was added also caused the `submit` button to be redrawn w/o the progress indicator. Bad jiji.